README.md

```diff
@@ -1,98 +1,144 @@
 # vcenter-connector
 
-vcenter-connector is an OpenFaaS event connector built to consume events from vCenter from VMware
+vcenter-connector is an OpenFaaS event connector built to consume events from vCenter from VMware vSphere.
 
-With this project your functions can subscribe to events generated by the changes in your vCenter installation - for instance a VM being created, turned on or deleted. This allows you to extend vCenter's functionality by writing functions to execute each time an event is fired. An example may be tagging a VM with the date it was last turned on or applying a tag showing which user made a change to an object. 
+With this project your functions can subscribe to events generated by the changes (i.e. events) in your vCenter installation - for instance a VM being created, turned on or deleted. This allows you to extend vCenter's functionality by writing functions to execute each time an event is fired. An example may be tagging a VM with the date it was last turned on or applying a tag showing which user made a change to an object. 
 
 ## Status
 
 This project uses the [OpenFaaS Connector SDK](https://github.com/openfaas-incubator/connector-sdk).
 
-The code is under active development and only suitable for early adopters. More examples and instructions for use in Kubernetes are being worked on.
+The code is under **active development** and only suitable for early adopters. More examples and instructions for use in Kubernetes are being worked on, including improved handling of secrets (accounts and passwords).
 
-## Example:
+**Note:** Currently only a pre-build Docker image is available (based on [PR#11](https://github.com/openfaas-incubator/vcenter-connector/pull/11)) and only [VM events](https://code.vmware.com/doc/preview?id=4206#/doc/vim.event.VmEvent.html) are supported.
```

Comment:
Are you sure about this?

If that's true (I wasn't sure that it was?) then put -> `are reportable` or similar